doc/docportal/use_scummvm/taskbar_integration.rst

```diff
@@ -4,18 +4,19 @@ Taskbar integration
 
 The taskbar integration has several features:
 
-- Adds an overlay icon in the taskbar/dock when running a game (Windows and macOS)
-- Updates a list of recently played games that can be started from the taskbar/dock (Windows and macOS).
-- Shows a progress bar when doing a mass add of games, and shows the number of games found at the end (Windows, macOS and Linux).
-- Shows an error state in the taskbar if an error occurs when running a game (Windows only).
+    - Adds an overlay icon in the taskbar/dock when running a game (Windows and macOS)
+    -  Updates a list of recently played games that can be started from the taskbar/dock (Windows and macOS).
+    -  Shows a progress bar when doing a mass add of games, and shows the number of games found at the end (Windows, macOS and Linux).
+    -  Shows an error state in the taskbar if an error occurs when running a game (Windows only).
 
 
 Setting up the icon overlay
 ================================
 
-Icons for many games can be found on our `Github repository <https://github.com/scummvm/scummvm-icons>`__.
+To download or update game icons, from the Launcher go to **Global Options**, go to the **Misc** tab, and click on the **Update Icons** button. See the :ref:`Update Icons <updateicons>` for more information.
```

Comment:
I will modify this when merging as currently the taskbar is unable to use the icons.dat file that we can download from the Global Options. So icons still need to be downloaded manually.

Reply:
Ahh thank you - I should've double checked this one, I just assumed!